data/tools/regal.yml

```diff
@@ -0,0 +1,19 @@
+name: Regal
+categories:
+  - linter
+tags:
+  - rego
+license: Apache License 2.0
+types:
+  - cli
+source: 'https://github.com/styrainc/regal'
+homepage: 'https://github.com/styrainc/regal'
+description: >-
+  Regal is a linter for the policy language Rego. Regal aims to catch bugs and mistakes
+  in policy code, while at the same time help people learn the language, best practices
```

Comment:
```suggestion
  in policy code, while at the same time helping people learn the language, best practices
```

Reply:
Ah, good catch! Fixed. I think I rewrote that sentence but missed that. I don't think there are any other static analysis tools for Rego at this point in time, but if I'm made aware of one later, I'll make sure to add it, or ask the author to do so. Thanks Matthias!